src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -735,7 +746,9 @@ export class MongoStorageAdapter implements StorageAdapter {
     schema: SchemaType,
     query: QueryType,
     readPreference: ?string,
-    hint: ?mixed
+    _estimate: ?boolean,
```

Comment:
What is the `_estimate` for? Is this part of this PR?

Reply:
yes
`count(
    className: string,
    schema: SchemaType,
    query: QueryType,
    readPreference?: string,
    estimate?: boolean,
    hint?: mixed,
    comment?: string
  ): Promise<number>;` 
in the StorageAdapter interface, there is estimate parameter, ignoring this parameter without adding it was causing the lint check to fail, I added estimate as the parameter in the count function in MongoStorageAdapter but this would also cause a warning about an unused parameter, renaming it to _estimate will ignore the warning.

Reply:
Why did lint fail if this PR does not seem to modify anything related to the estimate param, only adding the `_estimate`? Shouldn't lint have failed even before this PR?

Reply:
it should have failed only if `hint` was declared as object or string, but `hint` is type of mixed
this code was before the PR
StorageAdapter:
`  count(
    className: string,
    schema: SchemaType,
    query: QueryType,
    readPreference?: string,
    estimate?: boolean,
    hint?: mixed
  ): Promise<number>;`

MongoStorageAdapter:
`count(
    className: string,
    schema: SchemaType,
    query: QueryType,
    readPreference: ?string,
    hint: ?mixed
  )`

lint was assuming that `estimate` is `hint` because hint is of type mixed which is wrong, in my case if I changed the the type of `comment` to mixed the lint check would also pass but it wouldn't be correct because it would be assuming that `hint` is `comment`

Reply:
did you re run the workflows or that's just the previous one? On my side all tests passed on all mongoDb versions.

Reply:
Got it.